lib/node_modules/@stdlib/math/base/special/besselj0/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@stdlib/math/base/special/besselj0",
+  "version": "0.0.0",
+  "description": "Compute the Bessel function of the first kind of order zero.",
+  "author": {
+    "name": "The Stdlib Authors",
+    "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+  },
+  "contributors": [
+    {
+      "name": "The Stdlib Authors",
+      "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+    }
+  ],
+  "scripts": {},
+  "main": "./lib",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/stdlib-js/stdlib.git"
+  },
+  "homepage": "https://github.com/stdlib-js/stdlib",
+  "keywords": [
+    "stdlib",
+    "stdmath",
+    "mathematics",
+    "math",
+    "special",
+    "bessel",
+    "j0",
+    "number"
+  ],
+  "bugs": {
+    "url": "https://github.com/stdlib-js/stdlib/issues"
+  },
+  "dependencies": {},
+  "devDependencies": {},
+  "engines": {
+    "node": ">=0.10.0",
+    "npm": ">2.7.0"
+  },
+  "license": "Apache-2.0"
```

Comment:
As this package has ported Boost code, the license should be `Apache-2.0 AND BSL-1.0`. I have already fixed `sincos`.